tests/test_nitcorn.nit

```diff
@@ -17,8 +17,8 @@ import pthreads
 
 redef class Sys
 	var iface: String is lazy do
-		srand
-		return "localhost:{10000+20000.rand}"
+		var testid = "NIT_TESTING_ID".environ.to_i
+		return "localhost:{10000+testid}"
```

Comment:
Yeah, we should let the OS assign a port (cf #2087); we should also libify the nitcorn testing bits (cf #2098).
But I was tired of the recent race conditions so I just take the quick and dirty path.